### PR TITLE
Cart: tell webpack where the resources are located.

### PR DESCRIPTION
**Pull request by joshsadam**
Signed-off-by: Josh Adam <josh.adam@canada.ca>

## Description of changes
What issue where webpack did not know where to look for code split resources.

### CODE REVIEW

File: src/main/webapp/resources/js/pages/cart/index.js [hunk 1]
```diff
@@ -7,15 +7,22 @@ import {
...
   removeProjectFromCart,
-  removeSampleFromCart,
-  loadFullCart
+  removeSampleFromCart
 } from "../../redux/sagas/cart";
 import { Cart } from "./components/Cart";
 
+/*
+WEBPACK PUBLIC PATH:
+Webpack does not now what the servlet context path is.  To fix this, webpack exposed
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/cart/index.js]: Fixed typo in 1556436808882e14cc42f1ae7805b1c97d3b5c10

**Comment by apetkau** [comment]
It also looks like the **Export to Galaxy** page doesn't load properly the first time:

image

If you refresh or click **Export to Galaxy** again, it loads properly:

image

**Comment by apetkau** [comment]
It also looks like the Galaxy approval page doesn't display properly anymore:

image

Instead of displaying the above it just always auto-approves me.

**Comment by apetkau** [comment]
Everything works, thanks @joshsadam.

The Galaxy approval page must have been my fault with old cache in my browser. It works now.

**Pull request merged**